contrib/test_jenkins.sh

```diff
@@ -953,7 +953,8 @@ run_coverity() {
 			cov-format-errors --dir $cov_build --emacs-style
 			echo "not ok 1 Coverity Detected $nerrors failures # $cov_url" >> coverity.tap
 		else
-		echo "ok 1 Coverity found no issues" >> coverity.tap
+			echo "ok 1 Coverity found no issues. deleting the coverity directory" >> coverity.tap
```

Comment:
"deleting the coverity directory" should not be printed to coverity.tap, only to log file